### ISSUE TITLE
getErrorLocation

### ISSUE THREAD

**Issue by awgul** [cferdinandi/bouncer#46]
Hey,

This code here is problematic.
Why?
What if a field doesn't have a sibling?
I have just one field in a container and that's all and so because of that I get error because there is no sibling.

```
      // If the message should come after the field
      if (settings.messageAfterField) {
        return target.nextSibling;
      }
```

and than you use
`field.parentNode.insertBefore(error, location);`

and it should be like selecting `insertBefore` or `insertAfter`

Best,
Tom

**Comment by awgul** [comment]
Sorry, but there is no `insertAfter` :)
But I hope you understand what I mean here.
So if there is no sibling we could use `appendAfter` and in this case it would be like `insertAfter` because there is only one element.

**Comment by awgul** [comment]
Now when there is no sibling and i want to `messageAfterField=true` I get error
`Cannot read property 'parentNode' of null`, because no sibling.

**Comment by cferdinandi** [comment]
Please provide [a reduced test case](https://css-tricks.com/reduced-test-cases/).

**Comment by awgul** [comment]
Hey,

Have a look:
https://codepen.io/tawg/pen/MNPpLw

It doesn't create an error if there is some sibling, it works fine when there is a text node.
But in this case there is no white space after the input field.

**Comment by cferdinandi** [comment]
Nice find! I need to create an empty text node after the element if there isn't a sibling, it would seem.

I'll add this to `getErrorLocation()` (tested and it works)

```js
// If the message should come after the field
if (settings.messageAfterField) {

	// If there's no next sibling, create one
	if (!target.nextSibling) {
		target.parentNode.appendChild(document.createTextNode(''));
	}

	return target.nextSibling;
	
}
```

**Comment by cferdinandi** [comment]
Fixed in v1.4.6

**Comment by awgul** [comment]
Great, thank you.
